### DISCUSSION TITLE
How to move Search, Account, Wishlist and Minicart

### DISCUSSION THREAD

**Discussion by fvlasie** [breezefront/community#125] [1. Help] [answered]
Hello!

I am really excited to start using the Breeze Evolution theme. I have a question about customising the header. I would like to move the search, account, wishlist, minicart block into the area above navigation. I tried editing the default.xml but I could not get it to do what I wanted.

![Screenshot 2023-05-07 at 1 07 02 PM](https://user-images.githubusercontent.com/33334298/236700670-7ca77fbb-66f5-4244-b660-a219991b58d7.png)

I tried adding these to default.xml but it is not working (it is probably obvious but I am new to this! 😅 )

```
<move element="block.search" destination="panel-wrapper" after="header.panel.info"/>
<move element="switcher.account" destination="panel-wrapper" after="block.search"/>
<move element="switcher.wishlist" destination="panel-wrapper" after="switcher.account"/>
<move element="minicart" destination="panel-wrapper" after="switcher.wishlist"/>
```

**Comment by fvlasie** [comment]
I managed to get the search and cart elements moved up using:
```
	<move element="top.search" destination="header.panel"/>
        <move element="minicart" destination="header.panel"/>
```
But I cannot find the name of the account and wishlist elements....

**Comment by fvlasie** [comment, marked as answer]
I found the answer. It needs to be this:
```
	<move element="top.search" destination="header.panel"/>
        <move element="minicart" destination="header.panel"/>
        <move element="header.wishlist" destination="header.panel"/>
        <move element="header.account" destination="header.panel"/>
```
However, now these items disappear into the menu in mobile view!

**Reply by vovayatsyuk** [reply]
We hide top panel on mobile view. You can disable this feature using less variable:

```
@header-panel__slideout: false;
```

Source: https://breezefront.com/docs/header

**Reply by fvlasie** [reply]
Thank you!
